CHANGES.rst

```diff
@@ -23,6 +23,10 @@ New Features
   - Added ``PrecessedGeocentric`` frame, which is based on GCRS, but precessed
     to a specific requested mean equinox. [#3758]
 
+  - `SphericalRepresentation` now has a `._unitrep` class attribute to specify 
+    an equivalent UnitSphericalRepresentation. This allows subclasses of 
+    representations to pair up correctly.
```

Comment:
I did not know where best to put it in the change log, the way I have written this PR does assume that all `SphericalRepresentation` classes will have a `_unitrep` attribute now, so any authors will need to update.

Should I add a catch in the `BaseCoordinateFrame.__init__` for non-updated subclasses, or are the SunPy ones the only subclasses in existence?!